t/Renard/Curie/Component/ScrollZoom.t

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env perl
+
+use Test::Most;
+
+use lib 't/lib';
+use Renard::Curie::Setup;
+use Renard::Curie::App;
+use Renard::Curie::Model::Document::PDF;
+use CurieTestHelper;
+use Renard::Curie::Types qw(Int InstanceOf);
+use Function::Parameters;
+no strict;
+
+my $cairo_doc = CurieTestHelper->create_cairo_document;
+
+fun Scroll_Event( (InstanceOf['Renard::Curie::App']) $app) {
+	my $event = Gtk3::Gdk::Event->new('scroll');
+	$event->state( 'control-mask' );
+	$event->direction('down');
+	$app->page_document_component->signal_emit( 'scroll_event' => $event );
+}
+
+subtest 'Check that ctrl+scroll-up zooms into the page' => fun {
+	my ( $app, $page_comp ) = CurieTestHelper->create_app_with_document($cairo_doc);
+
+	print STDERR $page_comp->zoom_level . "\n";
```

Comment:
a better way of printing out values for tests is to the `note` function. It's part of the functions brought in by [`Test::Most`](https://p3rl.org/Test::Most).

If you want to print out data structures, I also recommend [`Data::Printer`](https://p3rl.org/Data::Printer).